vsintegration/src/FSharp.Editor/LanguageService/FSharpProjectOptionsManager.fs

```diff
@@ -209,58 +221,64 @@ type private FSharpProjectOptionsReactor (workspace: VisualStudioWorkspaceImpl,
                     ProjectId = Some(projectId.ToFSharpProjectIdString())
                     SourceFiles = projectSite.CompilationSourceFiles
                     OtherOptions = otherOptions
-                    ReferencedProjects = referencedProjects
+                    ReferencedProjects = referencedProjects.ToArray()
```

Comment:
Why the `ToArray`?

Reply:
referencedProjects is a ResizeArray, I have to do ToArray here for ReferencedProjects field.